test/contracts/managers/assetPairManagerV2.spec.ts

```diff
@@ -0,0 +1,1735 @@
+
+    feeCalculator = await feeCalculatorHelper.deployPerformanceFeeCalculatorAsync(
+      core.address,
+      oracleWhiteList.address,
+      ether(.4),
+      ether(.05)
+    );
+    feeCalculatorWhiteList = await coreHelper.deployWhiteListAsync([feeCalculator.address]);
+
+    rebalancingFactory = await coreHelper.deployRebalancingSetTokenV3FactoryAsync(
+      core.address,
+      rebalancingComponentWhiteList.address,
+      liquidatorWhiteList.address,
+      feeCalculatorWhiteList.address,
+    );
+
+    await core.addFactory.sendTransactionAsync(rebalancingFactory.address, { from: deployerAccount });
+    await blockchain.increaseTimeAsync(new BigNumber(2));
+    await core.addFactory.sendTransactionAsync(rebalancingFactory.address, { from: deployerAccount });
+    await erc20Helper.approveTransfersAsync(
+      [wrappedBTC, wrappedETH],
+      transferProxy.address
+    );
+
+    quoteAssetCollateral = await protocolHelper.createSetTokenAsync(
+      core,
+      factory.address,
+      [wrappedBTC.address],
+      [new BigNumber(1)],
+      new BigNumber(10 ** 10),
+    );
+
+    baseAssetCollateral = await protocolHelper.createSetTokenAsync(
+      core,
+      factory.address,
+      [wrappedETH.address],
+      [new BigNumber(33554432)],
+      new BigNumber(10 ** 6),
+    );
+
+    allocator = await managerHelper.deployBinaryAllocatorMockAsync(
+      baseAssetCollateral.address,
+      quoteAssetCollateral.address,
+      baseAssetCollateralValue,
+      quoteAssetCollateralValue,
+    );
+
+    [trigger] = await managerHelper.deployTriggerMocksAsync(1, [false]);
+
+    await oracleHelper.addAuthorizedAddressesToOracleProxy(
+      ethOracleProxy,
+      [allocator.address, liquidator.address, feeCalculator.address]
+    );
+
+    await oracleHelper.addAuthorizedAddressesToOracleProxy(
+      btcOracleProxy,
+      [allocator.address, liquidator.address, feeCalculator.address]
+    );
+  });
+
+  afterEach(async () => {
+    blockchain.revertAsync();
+  });
+
+  describe('#constructor', async () => {
+    let subjectCore: Address;
+    let subjectAllocator: Address;
+    let subjectTrigger: Address;
+    let subjectUseBullishAllocation: boolean;
+    let subjectAllocationDenominator: BigNumber;
+    let subjectBullishBaseAssetAllocation: BigNumber;
+    let subjectSignalConfirmationMinTime: BigNumber;
+    let subjectSignalConfirmationMaxTime: BigNumber;
+    let subjectLiquidatorData: Bytes;
+    let subjectCaller: Address;
+
+    beforeEach(async () => {
+      subjectCore = core.address;
+      subjectAllocator = allocator.address;
+      subjectTrigger = trigger.address;
+      subjectUseBullishAllocation = true;
+      subjectAllocationDenominator = new BigNumber(100);
+      subjectBullishBaseAssetAllocation = new BigNumber(100);
+      subjectSignalConfirmationMinTime = ONE_HOUR_IN_SECONDS.mul(6);
+      subjectSignalConfirmationMaxTime = ONE_HOUR_IN_SECONDS.mul(12);
+      subjectLiquidatorData = ZERO_BYTES;
```

Comment:
Can we make this something not `ZERO_BYTES` just to be sure it isn't set. Shouldn't affect the ability to successfully rebalance cause the current liquidator just ignores the `liquidatorData` parma